Library/Homebrew/shims/super/cc

```diff
@@ -353,19 +354,20 @@ class Cmd
   end
 
   def ldflags_linux(args)
+    versioned_glibc_dep = (@deps & ["glibc@2.13", "glibc@2.17"]).first
```

Comment:
And even more: same as above. Given how many times we're hardcoding these versions definitely feel like we should be using e.g. a consistent constant and a regex.